Sunrise/Helpers/AppRouting.swift

```diff
@@ -71,4 +71,16 @@ class AppRouting {
         homeNavigationController.popToRootViewControllerAnimated(true)
     }
 
+    /**
+        Switches to the account tab, and presents reservation overview view controller.
+    */
+    static func showReservationWithId(reservationId: String) {
+        guard let tabBarController = tabBarController, ordersNavigationController = tabBarController.viewControllers?[2] as? UINavigationController,
+                ordersViewController = ordersNavigationController.viewControllers.first as? OrdersViewController else { return }
+
+        tabBarController.selectedIndex = 2
```

Comment:
A bit unrelated to this PR: Maybe it's better to define a constant for the tab bar indexes? Then we can change it in one place, e.g. if we re-add the category tab.